packages/react-core/src/components/Truncate/examples/Truncate.md

```diff
@@ -24,7 +26,8 @@ import { Truncate } from '@patternfly/react-core';
 ```js
 import React from 'react';
 import { Truncate } from '@patternfly/react-core';
-<div className="pf-v5-c-truncate--example">
+import truncateStyles from '@patternfly/react-styles/css/docs/components/Truncate/examples/Truncate';
+<div className={truncateStyles.truncateExample}>
```

Comment:
Same here and the other 2 places in this file. Don't need to import `truncateStyles` and just call this something generic.